examples/boto3/topic_with_endpoint.py

```diff
@@ -15,7 +13,7 @@
     topic_name = sys.argv[1]
     region_name = ""
```

Comment:
`region_name` is not used. please remove

Reply:
Thanks for this.